shared/js/background/email-utils.es6.js

```diff
@@ -83,6 +86,65 @@ export const getAddresses = () => {
     }
 }
 
+function sendPixelRequest (pixelName, params) {
+    const randomNum = Math.ceil(Math.random() * 1e7)
+
+    const searchParams = new URLSearchParams(Object.entries(params))
+
+    const url = getURL(pixelName) + `?${randomNum}&${searchParams.toString()}`
+
+    load.url(url)
+}
+
+function currentDate () {
+    return new Date().toLocaleString('en-CA', {
+        timeZone: 'America/New_York',
+        dateStyle: 'short'
+    })
+}
+
+const getFullPixelName = (name, browserName) => {
+    return `${name}_${browserName.toLowerCase()}`
+}
+
+const fireAddressUsedPixel = (pixel) => {
+    const browserInfo = parseUserAgentString()
+    const browserName = browserInfo?.browser?.toLowerCase() ?? 'unknown'
+    if (browserName === 'firefox') return
```

Comment:
@jonathanKingston @sammacbeth Do you think it's enough? Do we have a way to eliminate this code entirely for Firefox? I don't see anything like this, am I missing anything? Not sure if it's worth it to make the build process more complex.

Reply:
We can try like this and see if that is enough for the Mozilla reviewers. I think also defining this at the top of the file as a constant would make it clearer, i.e.:
```javascript
const pixelsEnabled = utils.getBrowserName() !== 'moz'
```